openwisp_radius/templates/html_password_reset_email.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'openwisp_utils/email_template.html' %}
+{% load i18n %}
+{% load l10n %}
+
+{% block mail_header %}
+{{ block.super }}
+<div class="subject">
+{% blocktrans %}Password reset on {{ site_name }}!{% endblocktrans %}
+</div>
+{% endblock %}
```

Comment:
The generated output doesn't look right:

```html
      <div class="logo-container">
        <img src="" alt="Logo" id="logo" class="logo">
      </div>
      <div class="subject">
        
      </div>
      
<div class="subject">
Password reset on example.com!
</div>
```

I would remove the exclamation mark.

Can't you pass this string in the context argument of `send_email` above?